server/services/auth.js

```diff
@@ -0,0 +1,52 @@
+import jwt from 'jsonwebtoken';
+
+import config from '../../config/env';
+import User from '../models/user';
+import userService from '../services/user';
+
+function loginUser(email) {
+  return new Promise((resolve) => {
+    User.getByEmail(email)
```

Comment:
`User.getByEmail()` returns a promise already, you can strip out most of this function